src/main/java/PageObjects/FuelsAndBiocomponentsRequirementsPage.java

```diff
@@ -25,14 +26,26 @@ public FuelsAndBiocomponentsRequirementsPage(WebDriver driver) {
         webDriverWait = new WebDriverWait(driver, Duration.ofSeconds(5));
     }
 
-    public FuelsAndBiocomponentsRequirementsPage clickOnDieseBtn(){
+    public FuelsAndBiocomponentsRequirementsPage clickOnDieseBtn() {
         WebElement requirementsTab = webDriverWait.until(ExpectedConditions.presenceOfElementLocated(dieselFuelsRequirementsBtn));
         requirementsTab.click();
         return new FuelsAndBiocomponentsRequirementsPage(driver);
     }
+
     public void scrollToFlashPointTableValue() {
         webDriverWait.until(ExpectedConditions.presenceOfElementLocated(flashPointTableValue));
         WebElement flashPointValue = driver.findElement(flashPointTableValue);
         ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", flashPointValue);
+        Actions actions = new Actions(driver);
+        actions.moveToElement(flashPointValue).doubleClick(flashPointValue).build().perform();
+    }
+
+    public boolean isFlashPointCorrect() {
+        boolean isFlashPointCorrect = false;
+        WebElement flashPoint = driver.findElement(By.xpath(".//div[@class='col-8 builder-container__content']/div[@id='faq-simple']/div[2]/div[@id='collapse2']//table//tr[8]"));
+        if (flashPoint.getText().contains("55,0")) {
+            isFlashPointCorrect = true;
+        }
```

Comment:
You can simplify this whole method to one line by just returning:
return flashPoint.getText().contains("55,0")